jdk/make/Images.gmk

```diff
@@ -163,22 +163,8 @@ ALL_BIN_LIST := $(call CacheFind, $(JDK_OUTPUTDIR)/bin)
 # Prevent sjavac from entering the images.
 ALL_BIN_LIST := $(filter-out %/sjavac, $(ALL_BIN_LIST))
 
-ifeq ($(OPENJDK_TARGET_OS), windows)
-  # On windows, the libraries are in the bin dir, only filter out debuginfo files
-  # for executables. "java" is both a library and executable.
-  ALL_BIN_EXEC_FILES := $(filter-out java.exe, $(notdir $(filter %.exe, $(ALL_BIN_LIST))))
-  ALL_BIN_DEBUG_FILTER := $(addprefix %, $(patsubst %.exe, %.debuginfo, $(ALL_BIN_EXEC_FILES)) \
-      $(patsubst %.exe, %.diz, $(ALL_BIN_EXEC_FILES))) %.pdb
-  ALL_BIN_LIST := $(filter-out $(ALL_BIN_DEBUG_FILTER), $(ALL_BIN_LIST))
-endif
-
-# Filter out files in .dSYM folders on macosx.
-ifeq ($(OPENJDK_TARGET_OS), macosx)
-  ALL_BIN_LIST := $(foreach file, $(ALL_BIN_LIST),$(if $(findstring .dSYM/,$(file)),,$(file)))
-endif
```

Comment:
Is removing these `macosx` JCL patch intentional?

Reply:
Yes, those files will still be removed by the [build script](https://github.com/adoptium/temurin-build/blob/master/sbin/build.sh#L770).